Documentation/design-docs/hw-intrinsics.md

```diff
@@ -0,0 +1,81 @@
+Implementation of Hardware Intrinsics in CoreCLR
+================================================
+This document describes the implementation of hardware intrinsics in CoreCLR.
+For information about how the intrinsic APIs are designed, proposed and approved,
+see https://github.com/dotnet/designs/blob/master/accepted/platform-intrinsics.md.
+
+In discussing the hardware intrinsics, we refer to the target platform, such as X86 or Arm64, as the "platform" and each set of extensions that are implemented as a unit (e.g. AVX2 on X64 or Simd on Arm64) as an "ISA".
+
+There is a design document for the Arm64 intrinsics: https://github.com/dotnet/coreclr/blob/master/Documentation/design-docs/arm64-intrinsics.md. It should be updated to reflect current (and ongoing) progress.
+
+## Overview
+
+The reference assemblies for the hardware intrinsics live in corefx, but all of the implementation is in the coreclr repo:
+
+* The C# implementation lives in coreclr/src/shared/System/Runtime/Intrinsics. These are little more than skeleton methods that usually invoke themselves recursively (see Indirect invocation support below).
```

Comment:
I see dotnet/coreclr : src\System.Private.CoreLib\shared\System\Runtime\Intrinsics and dotnet/corefx : src\Common\src\CoreLib\System\Runtime\Intrinsics. Which are you referring to, or both?

Reply:
They should be the same, since those are the sources that are mirrored between the two repositories.

Reply:
That would be useful information to include. (I mostly was pointing out that the documented path didn't correspond to either actual location)